buf/registry/owner/v1beta1/user_service.proto

```diff
@@ -82,6 +82,23 @@ message ListUsersRequest {
   // If this is empty, all Users for all Organizations are listed, but this functionality
   // is limited to Users with the necessary permissions.
   repeated OrganizationRef organization_refs = 3;
+  message Filter {
+    // Filter by the type of the User.
+    repeated UserType type = 1 [(buf.validate.field).enum.defined_only = true];
+    // Filter by the state of the User.
+    repeated UserState state = 2 [(buf.validate.field).enum.defined_only = true];
+  }
+  // The filter to apply.
+  Filter filter = 4;
+  enum OrderDirection {
```

Comment:
Why would we add this specifically for the ListUsers endpoint?

This is incongruent with the rest of the registry API - if we want to make such a change, propose something/a pattern that universally applies

Reply:
Created a shared enum `Order` in a new package `buf.registry.v1beta1` to apply the enum on each list request message.